frontend/src/components/course/components/edit-category-modal/edit-category-modal.tsx

```diff
@@ -1,3 +1,4 @@
+import { AppRoute, PaginationDefaultValue } from 'common/enums/enums';
```

Comment:
move this component to the `frontend/src/component/common`
since we need it in several places